src/tools/pub/cmd/pub-global.md

```diff
@@ -90,33 +99,83 @@ package again.
 
 ## Running a script
 
-You can run a script from an activated package explicitly using
-`pub global run`, or you can add it to your PATH so that you can run it
-directly at the command line.
+You can directly run a script from an activated package from the
+command line. If you are unable to run the script directly,
+you can also use `pub global run`.
+
+### Running a script from your PATH
+
+To run a script directly from the command line, add the `bin` file
+for the [system cache](/tools/pub/glossary#system-cache) to your path.
+
+For example, say you've activated the Stagehand script,
+but you still can't run the command:
+
+{% prettify sh %}
+$ pub global activate stagehand
+$ stagehand
+-bash: stagehand: command not found
+{% endprettify %}
+
+Verify that the `bin` directory for the system cache is in your path.
+The following path, on macOS, includes the system cache.
+
+{% prettify sh %}
+$ echo $PATH
+[[highlight]]/Users/<user>/.pub-cache/bin[[/highlight]]:/Users/<user>/homebrew/bin:/usr/local/bin:/usr/bin:/bin
+{% endprettify %}
+
+If this directory is missing from your path,
+locate the file for your platform and add it.
+
+|-------------------+---------------------------|
+|      Platform     |      Cache location       |
+|-------------------|---------------------------|
+| macOS or Linux | `~/.pub-cache/bin`        |
+| Windows<sup><strong>*</strong></sup> | `%APPDATA%\Pub\Cache\bin` |
+{:.table .table-striped}
+
+<sup><strong>*</strong></sup> The exact location of the system cache
+may vary for different versions of Windows.
+
+You can now directly invoke the command:
+
+{% prettify sh %}
+$ mkdir angular-project
```

Comment:
Let's use an underscore-separated name here. (I think this dash-separated name works, but it's not our convention.)